### ISSUE TITLE
multipart/form-data

### ISSUE THREAD

**Issue by DarioMagniPLRM** [hosseinmd/swagger-typescript#53]
Hi, I am using your library and can confirm that it works very well!

Unfortunately, I am not able to use some of the generated services and have not found any examples to help me.
# Example
## in services.ts
```ts
/**
 *
 * Upload file
 */
export const postFiles = (
  requestBody: {
    /**
     *
     * - Format: binary
     */
    file?: string
  },
  queryParams: PostFilesQueryParams,
  configOverride?: AxiosRequestConfig
): Promise<SwaggerResponse<any>> => {
  return Http.postRequest(postFiles.key, queryParams, objToForm(requestBody), undefined, overrideConfig(_CONSTANT1, configOverride))
}

/** Key is end point string without base url */
postFiles.key = '/files/'
```
## in types.ts
```ts
export interface PostFilesQueryParams {
  targetFolder: string
  protected?: boolean
}
```
## in config.ts (for QueryParams) you need to set
```ts
encode: params => qs.stringify(params, { arrayFormat: 'brackets' })
```
## in my application
assuming we have the following data:
```ts
const targetFolder = "upload/test/"
const filename = "test.png"
const fileBuffer = ... // the buffer of the file
```
# The question
**What is the correct way to call postFiles() to upload a file?**

this solution
```ts
const myForm  = {
    file: fileBuffer.toString('binary')
  }
const headerParam: PostFilesQueryParams = {
    targetFolder, 
    protected: false
  }
const result = await postFiles(myForm, headerParam) 
```
and many others like it, also using libraries for creating FormData, which I have tried do not work.

Can you help me by posting an example?

**Comment by hosseinmd** [comment]
This is an example of web input file upload
```js
<input
  onChange={(e) => {
    const file = e.target.files?.[0];

    postFile({ file: file as any });
  }}
  type="file"
/>;
```

**Comment by hosseinmd** [comment]
fixed on v6.6.1
Now you could put your own formData as body
```ts
postFile(new FormData() as any);
```

**Comment by hosseinmd** [comment]
if you are on nodejs use form-data npm package

**Comment by DarioMagniPLRM** [comment]
thank you! I will try it out. Can you post version 6.6.1 on npmjs?

**Comment by DarioMagniPLRM** [comment]
Thank you!
I can confirm that the following code, using [v6.6.1](https://github.com/hosseinmd/swagger-typescript/commit/7048bf4d2fda727de2ab89dcf0f64b0ba1b34d7c) and [form-data](https://github.com/form-data/form-data), is working  
```ts
import FormData from 'form-data'
import { postFiles } from ... // api services
import { PostFilesQueryParams } from .... // api types

// TEST DATA
const targetFolder = "upload/test/"
const filename = "test.png"
const fileBuffer = ... // the buffer of the file

// FILE UPLOAD FUNCTION
const formData = new FormData()
  formData.append('file', fileBuffer, { filename });

const headerParam: PostFilesQueryParams = {
    targetFolder,
    protected: false
}

const result = postFiles(formData as any, headerParam) // fixed in v6.6.1
console.log(result)
```
